test/__snapshots__/Button.test.tsx.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
No idea... Could you tell me the difference between Button.test.tsx vs Button.test.tsx.snap

Reply:
It's just a spanshot generated by jest. It's just to catch unexpected DOM changes.